crates/mvr-cli/src/lib.rs

```diff
@@ -438,7 +439,8 @@ pub async fn build_lock_files(
         let root_name_from_source = parse_source_package_name(&move_toml_content)?;
         let lock_with_root =
             insert_root_dependency(&move_lock_content, &root_name_from_source, &git_info)?;
-        lock_files.push(lock_with_root);
+
+        lock_files.push(lock_with_root.to_string());
```

Comment:
I think we decided to move the local smashing back into `insert_root_dependency`, so we can revert this change